### ISSUE TITLE
Project management: create and manage a Gitcoin bounty for zk-transfer-web

### ISSUE THREAD

**Issue by cryptogoth** [invisible-college/democracy#47]
## Background

This task is part of a larger project to create an interactive web demo / playground for users to learn about zero-knowledge assets and gain intuition about them by directly transacting with the contracts on-chain.

You can view a demo-in-progress at https://zk-transfer.netlify.com

## This Task

This is a project management meta-issue: you will be reading the high-level requirements of this mockup in a previous design issue #36 and the main mockup:

https://www.figma.com/file/KEV19jwlvgd3m9lexQx9KL/Confidential-Transfer-Demo?node-id=0%3A1

Based on the difference between this mockup and the current demo https://zk-transfer.netlify.com, 
you'll be asked to carve out a small, well-specified front-end development task similar to #43 or #41 .

It's called a meta-issue because it's an issue (assigned to you) that will create another issue (that you'll assign to a dev from Gitcoin).

You will be given a budget of 25 DAI, which I'll contribute to the bounty and send you the fees for. I'll ask you to post a new bounty on Gitcoin, approve workers who apply for the task, discuss with them and me the completion of the task in Gitter or on this thread, and then finally sign off, pay out the bounty, and rate the developer.

You don't have to do any coding or design yourself, only understanding how they work together, how to communicate with and support a developer, and how to create and manage bounties is a key skill.

If successful, this task will repeat, until the project is complete.

## How to Work on this Task

* Join [our gitter](https://gitter.im/invisible-college/democracy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and say hi. Let's talk about your ideas for an issue / bounty, send links to your previous project, and I'll work with you to create a two-sentence proposal. We'll use #43 or #41 as examples.

* Apply to work on this Gitcoin bounty, and I'll accept.

* Once you're approved, create the issue at https://github.com/invisible-college/democracy/issues
You can use the text of #43 or #41 as a boilerplate.

* Submit a link to your *new issue* on Gitcoin, and I'll review and work together with you to shape it.

* When we're happy, you'll create a new Gitcoin bounty for your *new issue* and I'll fund it with 25 DAI, with a deadline of one week from now. We'll invite some previous developers.

* You'll review workers who apply, and we'll accept one together, giving first dibs to previous developers who have successfully completed a bounty. We'll ask that developer to join us on Gitter.

* As the deadline approaches, you'll request updates and feedback from the developer, and I'll be available on Gitter to help unblock or answer questions.

* When the bounty is finish, you'll pay out the developer bounty on Gitcoin, and then I'll pay out this bounty to you. Looking forward to working with you.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 25.0 DAI (25.0 USD @ $1.0/DAI)  attached to it as part of the invisible-college fund.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/invisible-college/democracy/47/3927).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/invisible-college/democracy/47/3927).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $74,343.01 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 5 days, 9 hours from now.
Please review their action plans below:


**1) [sirlupinwatson1](https://gitcoin.co/sirlupinwatson1) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/invisible-college/democracy/47/3927?mutate_worker_action=approve&worker=sirlupinwatson1) | [reject worker](https://gitcoin.co/issue/invisible-college/democracy/47/3927?mutate_worker_action=reject&worker=sirlupinwatson1))_.**

Hi , i would be interested to this bounties. 

The plan is pretty straight forward since everything is well explained.

Thanks!

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/invisible-college/democracy/47/3927).

**Comment by Sirlupinwatson1** [comment]
Thanks for the approval @cryptogoth !

Much appreciated. 
---------------------------

I did send you a message on [#democracy channel](https://gitter.im/invisible-college/democracy)
and [private message](https://gitter.im/cryptogoth)

<img src="http://buildinternet.s3.amazonaws.com/images/dribbble-php/github-octacat.jpg" alt="See the source image" class=" nofocus" tabindex="0" aria-label="See the source image" data-bm="4">

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 25.0 DAI  (25.0 USD @ $1.0/DAI) attached to this issue has been **cancelled** by the bounty submitter__

 
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $75,422.52 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by cryptogoth** [comment]
This issue was an experiment to try one way of breaking up project management, a key task in guiding any software project from concept / roadmap, through implementation and shipping, and then support / maintenance.

**Comment by cryptogoth** [comment]
Well, we'll keep trying :)
